java/java-binding/src/main/java/com/risingwave/java/binding/Binding.java

```diff
@@ -26,17 +26,15 @@ public class Binding {
         }
     }
 
+    static void ensureInitialized() {}
```

Comment:
The native method `iteratorNewHummock` is moved to the `HummockIterator` class. We should ensure that this `Binding` class has been initialized before we use the `HummockIterator` class. This method is called in the static block of `HummockIterator` to ensure that the `Binding` class is loaded and initialized, and otherwise the shared library won't be loaded when we use `HummockIterator` and causes error.